src/mongocxx/test/spec/unified_tests/assert.cpp

```diff
@@ -172,6 +172,24 @@ bool is_special(T doc) {
            string::to_string(doc.get_document().value.begin()->key()).rfind("$$", 0) == 0;
 }
 
+// The C driver returns different names than the spec test. This will return the
+// name as the C driver declares it.
+std::string get_alternate_name(const std::string& key) {
```

Comment:
```suggestion
bsoncxx::stdx::string_view get_alternate_name(bsoncxx::stdx::string_view key) {
```
Use `bsoncxx::stdx::string_view` to avoid redundant allocations.